notebooks/build/covid-19.ipynb

```diff
@@ -0,0 +1,4219 @@
+{
```

Comment:
<p><code>max_30d_hospital_confirmed_for_peak</code>  looks really high, both just in general and compared to <code>peak_icu_patients</code>  -- any idea what's going on with that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oughtinc/ergo/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>